packages/generator/src/parser/common/edmx-to-vdm.ts

```diff
@@ -133,36 +133,39 @@ function properties(
   complexTypes: VdmComplexType[],
   formatter: ServiceNameFormatter
 ): VdmProperty[] {
-  return entity.entityType.Property.filter(filterUnknownEdmTypes).map(p => {
-    checkCollectionKind(p);
-    const swaggerProp = entity.swaggerDefinition
-      ? entity.swaggerDefinition.properties[p.Name]
-      : undefined;
-    const instancePropertyName = formatter.originalToInstancePropertyName(
-      entity.entitySet.Name,
-      p.Name
-    );
-    const type = parseTypeName(p.Type);
-    const isComplex = isComplexType(type);
-    return {
-      originalName: p.Name,
-      instancePropertyName,
-      staticPropertyName: formatter.originalToStaticPropertyName(
+  return entity.entityType.Property.filter(filterUnknownEdmTypes)
+    .map(p => {
+      checkCollectionKind(p);
+      const swaggerProp = entity.swaggerDefinition
+        ? entity.swaggerDefinition.properties[p.Name]
+        : undefined;
+      const instancePropertyName = formatter.originalToInstancePropertyName(
         entity.entitySet.Name,
         p.Name
-      ),
-      propertyNameAsParam: applyPrefixOnJsConfictParam(instancePropertyName),
-      edmType: type,
-      jsType: propertyJsType(type) || complexTypeForName(type, complexTypes),
-      fieldType:
-        propertyFieldType(type) || complexTypeFieldForName(type, complexTypes),
-      description: propertyDescription(p, swaggerProp),
-      nullable: isNullableProperty(p),
-      maxLength: p.MaxLength,
-      isComplex,
-      isCollection: isCollection(p.Type)
-    };
-  });
+      );
+      const type = parseTypeName(p.Type);
+      const isComplex = isComplexType(type);
+      return {
+        originalName: p.Name,
+        instancePropertyName,
+        staticPropertyName: formatter.originalToStaticPropertyName(
+          entity.entitySet.Name,
+          p.Name
+        ),
+        propertyNameAsParam: applyPrefixOnJsConfictParam(instancePropertyName),
+        edmType: type,
+        jsType: propertyJsType(type) || complexTypeForName(type, complexTypes),
+        fieldType:
+          propertyFieldType(type) ||
+          complexTypeFieldForName(type, complexTypes),
+        description: propertyDescription(p, swaggerProp),
+        nullable: isNullableProperty(p),
+        maxLength: p.MaxLength,
+        isComplex,
+        isCollection: isCollection(p.Type)
+      };
+    })
+    .filter(filterEnumProperties);
```

Comment:
I assume you only added the filter at the end 🤦

Reply:
Haha, yes, but it changes the formatting.